packages/forms/backend/sirius-components-forms/src/main/java/org/eclipse/sirius/components/forms/components/FormComponent.java

```diff
@@ -31,10 +31,28 @@
 public class FormComponent implements IComponent {
 
     /**
-     * The variable name used to store a {@link WidgetIdCounter} in the {@link VariableManager} of the
-     * {@link FormComponent}.
+     * The name of the variable used to store the id of the parent element in the {@link VariableManager} for a widget
+     * to compute its own id.
      */
-    public static final String WIDGET_ID_PROVIDER_COUNTER = "widgetIdProviderCounter";
+    public static final String PARENT_ELEMENT_ID = "parentElementId";
+
+    /**
+     * The name of the variable used to store the id of widget's description in the {@link VariableManager} for a widget
+     * to compute its own id.
+     */
+    public static final String CONTROL_DESCRIPTION_ID = "controlDescriptionId";
+
+    /**
+     * The name of the variable used to store the id of widget's description in the {@link VariableManager} for a widget
+     * to compute its own id.
+     */
+    public static final String TARGET_OBJECT_ID = "targetObjectId";
+
+    /**
+     * The name of the variable used to store the id of widget's description in the {@link VariableManager} for a widget
+     * to compute its own id.
+     */
+    public static final String WIDGET_LABEL = "widgetLabel";
 
```

Comment:
It's the same comment for the three variables

Reply:
Done.